### ISSUE TITLE
now the `createcollectionswagger` apis don't work properly

### ISSUE THREAD

**Issue by locustbaby** [qdrant/qdrant#1880]
now the `createcollectionswagger` apis don't work properly

## Current Behavior

I setuped a qdrant locally, and then I wanna to create a collection through https://ui.qdrant.tech/?v=v1.1.1 
Excuted but an error `{"status":{"error":"Format error in JSON body: data did not match any variant of untagged enum VectorsConfig at line 14 column 3"},"time":0.0}% `
![image](https://github.com/qdrant/qdrant/assets/21237232/e34e1c86-9b5a-4bf4-bbc1-f6c7e08c1d52)

## Steps to Reproduce

1.
2.
3.
4.

## Possible Solution

Found `quantization_config` was setted as "string", so I added a `default null ` for it, 
But a new error showed up which tells `hnsw_config.m` should be between [4, 10000]
![image](https://github.com/qdrant/qdrant/assets/21237232/19e425dd-5254-4e93-9c49-13a62ee98ba1)

Why  the `hnsw_config.m` should be between [4, 10000], the docs says it's ok  `m>=0`
![image](https://github.com/qdrant/qdrant/assets/21237232/aa9c0eb0-af68-49e8-846a-40efc35f0ece)

So I set the default `m` to `4`, again a new error :

```
{"status":{"error":"Service internal error: Can't init WAL: Custom { kind: InvalidInput, error: \"invalid segment capacity: 0\" }"},"time":0.287476417}%    
```   

After changed the `wal_capacity_mb` to `1`,  request success

My commit [here](https://github.com/locustbaby/qdrant/commit/3a4e251390358f8409b496948c0cf54a8fd14ec1)
`hnsw_config.m` is didn't involved cause I don't know if it's changed from `>=0` to `>=4`

**Comment by generall** [comment]
hi @locustbaby !

We fixed `m` validation in the https://github.com/qdrant/qdrant/releases/tag/v1.1.2 release. Could you please upgrade to this verion?

**Comment by locustbaby** [comment]
> hi @locustbaby !
> 
> We fixed `m` validation in the https://github.com/qdrant/qdrant/releases/tag/v1.1.2 release. Could you please upgrade to this verion?

Oh, I just tried it and it's working now.

And what about the `wal_capacity_mb` ? 
Docs says `>=0` but failed when use it
![image](https://github.com/qdrant/qdrant/assets/21237232/e3e81700-d6dc-4937-9164-518794d90a04)

**Comment by generall** [comment]
`wal_capacity_mb` is actually not expected to be 0. Will look into it. It definitely should not be a 500 error as well

**Comment by locustbaby** [comment]
> hi @locustbaby !
> 
> We fixed `m` validation in the https://github.com/qdrant/qdrant/releases/tag/v1.1.2 release. Could you please upgrade to this verion?

I see there is a #1733  here
May I know where is the binary cause I didn't see the binary in the release 
<img width="643" alt="image" src="https://github.com/qdrant/qdrant/assets/21237232/9fc9ba93-81cf-481e-a2d7-794917bcf02b">

**Comment by generall** [comment]
> May I know where is the binary cause I didn't see the binary in the release

We didn't merge it ti master yet. Will be in v1.2.0

**Comment by coszio** [comment]
[Binaries](https://github.com/qdrant/qdrant/releases/tag/v1.2.2) are now available since v1.2.x, as promised.

Closing this issue
